app/views/_container-statuses.html

```diff
@@ -0,0 +1,72 @@
+<!--
+  Expects the following variables:
+  containerStatuses
+  startDebugTerminal (optional)
+  detailed (optional)
+ -->
+
+<div ng-if="detailed">
+  <h4 row>
```

Comment:
So I was thinking we would just always show the init containers if they're running since they're the most interesting thing under status, then only show this status summary (with show details) when they all complete.

@jwforres opinion?